pkg/glanceapi/volumes.go

```diff
@@ -45,5 +46,7 @@ func getInitVolumeMounts(extraVol []glancev1.GlanceExtraVolMounts, svc []storage
 			vm = append(vm, vol.Mounts...)
 		}
 	}
+	_, secretConfig := glance.GetConfigSecretVolumes(secretNames)
+	vm = append(vm, secretConfig...)
```

Comment:
May be call that method here as well?

Reply:
see my previous comment about the reason I'd like to keep it here.